crates/sui-core/src/authority.rs

```diff
@@ -1800,10 +1798,19 @@ impl AuthorityState {
         let mut execution_lock = db.execution_lock_for_reconfiguration().await;
         self.revert_uncommitted_epoch_transactions(cur_epoch_store)
             .await?;
-        if self.enable_state_snapshot {
-            let _checkpointed_db_path = self.checkpoint_perpetual_db()?;
-            // TODO: Start a background task to persist live object set in
-            // checkpointed db to canonical storage format
+        if self.db_checkpoint_config.checkpoint_path.is_some()
```

Comment:
You can use `if let` so that latter you have the path

Reply:
done